README.md

```diff
@@ -154,53 +156,47 @@ In the following i provide an example:
 
 In your `MediaAdminController.php`:
 ```php
-    /**
-     * @return RedirectResponse
-     */
-    public function createGalleryAction(Request $request)
-    {
-        $idx = $request->query->get('idx');
-        $idx = json_decode($idx);
-
-        $galleryManager = $this->get('sonata.media.manager.gallery');
-
-        $gallery = $galleryManager->create();
-        $gallery->setName('Auto Created Gallery');
-        $gallery->setEnabled(false);
-        $gallery->setContext('default');
-
-        $mediaManager = $this->get('sonata.media.manager.media');
-        foreach ($idx as $id) {
-            $media = $mediaManager->find($id);
-
-            $galleryHasMedia = new GalleryHasMedia();
-            $galleryHasMedia->setGallery($gallery);
-            $galleryHasMedia->setMedia($media);
-            $gallery->addGalleryHasMedia($galleryHasMedia);
-        }
+public function createGalleryAction(Request $request, MediaManager $mediaManager, GalleryManager $galleryManager): RedirectResponse
```

Comment:
Did you test this is working? 

Maybe add use statements ?

Reply:
i mean for GalleryManager and MediaManager you need to create an alias like this before this works.

```yaml
  Sonata\MediaBundle\Entity\MediaManager:
        alias: sonata.media.manager.media

    Sonata\MediaBundle\Entity\GalleryManager:
        alias: sonata.media.manager.gallery

```